### PR TITLE
Adds "skipsynchronization" property into TrackedEntityAttribute - DHIS2-3300

### PR DESCRIPTION
**Pull request by huzvak**

### CODE REVIEW

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttribute.java [hunk 1]
```diff
@@ -424,10 +426,22 @@ public void setFormName( String formName )
...
     public Boolean isSystemWideUnique()
     {
         return isUnique() && !getProgramScope() && !getOrgunitScope();
     }
 
+    @JsonProperty
+    @JacksonXmlProperty( namespace = DxfNamespaces.DXF_2_0 )
+    public Boolean getSkipSynchronization()
+    {
+        return skipSynchronization != null ? skipSynchronization : false;
```
Comment (stian-sandvold) [inline dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttribute.java]: You should not do a null-check here.

false should be default(?) and during startup routines, any null values should be set to the default value. In other words at this point it should be either true or false, never null.

File: dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentity/hibernate/TrackedEntityAttribute.hbm.xml [hunk 2]
```diff
@@ -69,14 +70,16 @@
...
-    <property name="textPattern" type="jbTextPattern" column="textpattern" not-null="false"/>
+    <property name="textPattern" type="jbTextPattern" column="textpattern" not-null="false" />
 
     <property name="style" type="jbObjectStyle" column="style" />
 
     <property name="orgunitScope" />
 
     <property name="programScope" />
 
+    <property name="skipSynchronization" column="skipsynchronization" not-null="false" />
```
Comment (stian-sandvold) [inline dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentity/hibernate/TrackedEntityAttribute.hbm.xml]: when name == column, you don't need to specify column name.

File: dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttribute.java (hunk 1 above)
Comment (huzvak) [reply dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttribute.java]: Fixed

File: dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentity/hibernate/TrackedEntityAttribute.hbm.xml (hunk 2 above)
Comment (huzvak) [reply dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/trackedentity/hibernate/TrackedEntityAttribute.hbm.xml]: Explicit column name removed.

**Pull request merged**